R/run.R

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8-unix -*-
+
+library(here)
+library(tidyverse)
+options(scipen = 999)
+
+# Set envinronment for config file ----
+
+Sys.setenv(R_CONFIG_ACTIVE = "V2030_ve0_rm")
+
+# Create output folder ----
+
+dir.create(here("results", config::get("projected_scenario")),
+           showWarnings = FALSE)
+
+# Combine data from different Helmet-scenarios ----
+
+source(here("R", "combine_agent_data.R"), encoding = "utf-8")
+
+# Read themes and colors for plotting ----
+
+source(here("R", "colors.R"), encoding = "utf-8")
+source(here("R", "themes.R"), encoding = "utf-8")
+
+# Plot results ----
+
+# The analysis files should be self-contained i.e.
+# the order in which the files are ran does not matter
+
+analysis_files <- list.files(path = here("R/plotting"),
```

Comment:
Replace `here("R/plotting")` with `here("R", "plotting")`. It is slower to write but works better if the underlying OS changes.